bindings/nodejs/generated.d.ts

```diff
@@ -22,7 +22,7 @@
 
 /* auto-generated by NAPI-RS */
 
-export class ExternalObject<T> {
+export declare class ExternalObject<T> {
```

Comment:
I don't know why this changed, do we have different version of `napi`?

Reply:
it changed by npm audit ..will dig itto it

Reply:
Those files are auto-generated, so if there's an issue, we should raise it with napi-rs instead. Fixing it on our side will be overridden.

Reply:
fixed

Reply:
I'm guessing we need to revert changes over this generated file.

Reply:
will do this tomorrow good night for today~